packages/widgets/src/components/SearchBox/index.ts

```diff
@@ -0,0 +1,5 @@
+export { SearchBox } from './SearchBox';
+export { SearchBoxCrossIcon } from './SearchBoxCrossIcon';
+export { SearchBoxInput } from './SearchBoxInput';
+export { SearchBoxWrapper } from './SearchBoxWrapper';
+export { StoryComponent } from './stories/StoryComponent';
```

Comment:
We shouldn't export utility code from index.ts, e.g., stories, fixtures, tests, etc.